### ISSUE TITLE
With 2 carousels and browser width > 1100 center mode items fail to size properly

### ISSUE THREAD

**Issue by Naphier** [OwlCarousel2/OwlCarousel2#1475]
Using OwlCarousel2 2.1.6 and JQuery 3.0.0

If you set up 2 carousels - one with 3 items in the first carousel and the second carousel with 1 item, both carousels in center mode - then the carousels items fail to size correctly when the window is > 1100px. In Chrome dev mode this happens around 620px. 

It does not happen when the carousel with multiple items is set items > 3. But does seem to consistently happen with items <= 3.

Here's a demo page: [http://naplandgames.com/new/owlcarousel/sizing-issue.html](http://naplandgames.com/new/owlcarousel/sizing-issue.html)

Been banging my head on the wall thinking it was my CSS, but I'm pretty sure it's owl's calculations now.

Thanks!

EDIT: Same thing happens in FF. I just noticed that it seems to be due to the scroll bar. Seems like it's failing to include that in the calculation of the item widths when there's more than one carousel. Hope that helps track it down.

**Comment by Naphier** [comment]
It's definitely the vertical scrollbar. Does not require multiple carousels on a page. Does not require a certain set number of items. All of those were false indicators. It is only the scroll bar. I also downgraded to jQuery 2.2.4 to test and the result is the same. I've tried digging through the code to see how to fix, but I think the issue is that the scrollbar might not be present at the time the item widths are calculated. Addressing that is far beyond my js abilities, but without it this beautiful project looks not so beautiful :cry:
